### PR TITLE
release notes beta 5.24.0

### PR DESCRIPTION
**Pull request by ScottSuarez**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
 ## 5.24.0 (Unreleased)
+IMPROVEMENTS:
+* cloudrunv2: added `template.volumes.nfs` field to `google_cloud_run_v2_job` resource (beta) ([#7169](https://github.com/hashicorp/terraform-provider-google-beta/pull/7169))
```
Comment (slevenick) [inline CHANGELOG.md:3]:
```suggestion
* cloudrunv2: added `template.volumes.nfs` field to `google_cloud_run_v2_job` resource ([#7169](https://github.com/hashicorp/terraform-provider-google-beta/pull/7169))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+* storage: added `project_number` attribute to `google_storage_bucket` resource and data source ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+* storage: added ability to provide `project` argument to `google_storage_bucket` data source. This will not impact reading the resource's data, instead this helps users avoid calls to the Compute API within the data source. ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+
+BUG FIXES:
+* appengine: fixed a crash in `google_app_engine_flexible_app_version` due to the `deployment` field not being returned by the API ([#7175](https://github.com/hashicorp/terraform-provider-google-beta/pull/7175))
+* bigquery: fixed a crash when `google_bigquery_table` had a `primary_key.columns` entry set to `""` ([#7166](https://github.com/hashicorp/terraform-provider-google-beta/pull/7166))
+* compute: Added explicit update_encoder to `ComputeTargetHttpsProxy` and `ComputeRegionTargetHttpsProxy` resources. ([#7170](https://github.com/hashicorp/terraform-provider-google-beta/pull/7170))
```
Comment (slevenick) [inline CHANGELOG.md:14]: What does this fix?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+* dataflow: fixed an issue where the provider would crash when `enableStreamingEngine` is set as a `parameter` value in `google_dataflow_flex_template_job` ([#7160](https://github.com/hashicorp/terraform-provider-google-beta/pull/7160))
```
Comment (slevenick) [inline CHANGELOG.md:15]: snake_case the field name

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,19 @@
...
```
Comment (ScottSuarez) [inline CHANGELOG.md:15]:
```suggestion
* dataflow: fixed an issue where the provider would crash when `enable_streaming_engine` is set as a `parameter` value in `google_dataflow_flex_template_job` ([#7160](https://github.com/hashicorp/terraform-provider-google-beta/pull/7160))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,19 @@
...
+* storage: added `project_number` attribute to `google_storage_bucket` resource and data source ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+* storage: added ability to provide `project` argument to `google_storage_bucket` data source. This will not impact reading the resource's data, instead this helps users avoid calls to the Compute API within the data source. ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+
+BUG FIXES:
+* appengine: fixed a crash in `google_app_engine_flexible_app_version` due to the `deployment` field not being returned by the API ([#7175](https://github.com/hashicorp/terraform-provider-google-beta/pull/7175))
+* bigquery: fixed a crash when `google_bigquery_table` had a `primary_key.columns` entry set to `""` ([#7166](https://github.com/hashicorp/terraform-provider-google-beta/pull/7166))
+* compute: Added explicit update_encoder to `ComputeTargetHttpsProxy` and `ComputeRegionTargetHttpsProxy` resources. ([#7170](https://github.com/hashicorp/terraform-provider-google-beta/pull/7170))
```
Comment (ScottSuarez) [inline CHANGELOG.md:14]:
```suggestion
* compute: Fixed update scenarios on `ComputeTargetHttpsProxy` and `ComputeRegionTargetHttpsProxy` resources. ([#7170](https://github.com/hashicorp/terraform-provider-google-beta/pull/7170))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,19 @@
...
+* storage: added `project_number` attribute to `google_storage_bucket` resource and data source ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+* storage: added ability to provide `project` argument to `google_storage_bucket` data source. This will not impact reading the resource's data, instead this helps users avoid calls to the Compute API within the data source. ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+
+BUG FIXES:
+* appengine: fixed a crash in `google_app_engine_flexible_app_version` due to the `deployment` field not being returned by the API ([#7175](https://github.com/hashicorp/terraform-provider-google-beta/pull/7175))
+* bigquery: fixed a crash when `google_bigquery_table` had a `primary_key.columns` entry set to `""` ([#7166](https://github.com/hashicorp/terraform-provider-google-beta/pull/7166))
+* compute: Fixed update scenarios on `ComputeTargetHttpsProxy` and `ComputeRegionTargetHttpsProxy` resources. ([#7170](https://github.com/hashicorp/terraform-provider-google-beta/pull/7170))
```
Comment (ScottSuarez) [inline CHANGELOG.md:14]:
```suggestion
* compute: fixed update scenarios on `ComputeTargetHttpsProxy` and `ComputeRegionTargetHttpsProxy` resources. ([#7170](https://github.com/hashicorp/terraform-provider-google-beta/pull/7170))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,19 @@
...
+* storage: added `project_number` attribute to `google_storage_bucket` resource and data source ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+* storage: added ability to provide `project` argument to `google_storage_bucket` data source. This will not impact reading the resource's data, instead this helps users avoid calls to the Compute API within the data source. ([#7164](https://github.com/hashicorp/terraform-provider-google-beta/pull/7164))
+
+BUG FIXES:
+* appengine: fixed a crash in `google_app_engine_flexible_app_version` due to the `deployment` field not being returned by the API ([#7175](https://github.com/hashicorp/terraform-provider-google-beta/pull/7175))
+* bigquery: fixed a crash when `google_bigquery_table` had a `primary_key.columns` entry set to `""` ([#7166](https://github.com/hashicorp/terraform-provider-google-beta/pull/7166))
+* compute: fixed update scenarios on `ComputeTargetHttpsProxy` and `ComputeRegionTargetHttpsProxy` resources. ([#7170](https://github.com/hashicorp/terraform-provider-google-beta/pull/7170))
```
Comment (slevenick) [inline CHANGELOG.md:14]: Can you change these to the terraform resource names?

Comment (zli82016) [reply CHANGELOG.md]: Resolved

**Review by zli82016 — COMMENTED** [review]
LGTM

**Pull request merged**